src/WizardsWardrobe.lua

```diff
@@ -31,19 +39,22 @@ function WW.GetSetupsAmount()
 end
 
 function WW.LoadSetupAdjacent( direct, skipValidation )
+	local isOpen = WW.IsAnySceneShown()
+    if isOpen then return end
 	local zone = WW.selection.zone
 	local pageId = WW.selection.pageId
 	local newSetupId = WW.currentIndex + direct
 	if newSetupId > WW.GetSetupsAmount() then newSetupId = 1 end
 	if newSetupId < 1 then newSetupId = WW.GetSetupsAmount() end
+
+	-- WW.ChangeDisplayedName(zone, pageId,  index)
 	WW.LoadSetup( zone, pageId, newSetupId, false, skipValidation )
 end
 
 function WW.IsReadyToSwap()
 	return not IsUnitInCombat( "player" ) and not IsUnitDeadOrReincarnating( "player" )
 end
 
-local setupTask = async:Create( WW.name .. "SetupTask" )

```

Comment:
You removed this local declaration without adding a new one elsewhere. This means the variable is now global. As explained [here](https://www.esoui.com/forums/showthread.php?t=10790), you should avoid polluting the global namespace.